### ISSUE TITLE
[CI] Integrate with Fastlane

### ISSUE THREAD

**Issue by rafaelks** [RocketChat/Rocket.Chat.iOS#1039]
- [ ] Integrate with Fastlane;
- [ ] Generate TestFlight builds when merging to Develop;
- [ ] Generate TestFlight builds when merging to Beta (send to external testers);
- [ ] Generate TestFlight builds when merging to Master (send to external testers);
- [ ] Notify Rocket.Chat channel (`#ios-fastlane`) on build success/failures;

**Comment by cardoso** [comment]
Shouldn't we generate the TestFlight build *only* when merging to beta? And send to live App Store when merging to master?

**Comment by artrmz** [comment]
- Merging to develop should increment build number (and generate TestFlight build for internal testers)
- Merging to beta should increment version number (and generate TestFlight build for external testers)
- Merging to master should generate AppStore build.

In my opinion.

**Comment by rafaelks** [comment]
Yes, that makes sense guys. I guess we can submit the latest Beta with no problem at all. Thanks for your suggestions! 👍
